api-docs-slate/source/includes/_wallet_tx.md

```diff
@@ -248,15 +281,26 @@ end="1506909154"
 ```
 
 ```shell--curl
-curl $url/wallet/$id/tx/range?start=$start
+curl $walleturl/$id/tx/range?start=$start
 ```
 
 ```javascript
-const wallet = WalletClient.wallet(id);
+const {WalletClient} = require('bclient');
+const {Network} = require('bcoin');
+const network = Network.get('regtest');
+
+const walletOptions = {
+  network: network.type,
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const walletClient = new WalletClient(walletOptions);
+const wallet = walletClient.wallet(id);
 
 (async () => {
-  const response = await wallet.getRange(account, {start: start, end: end});
-  console.log(response);
+  const result = await wallet.getRange(account, {start: start, end: end});
```

Comment:
Did this work for you?

Reply:
yeah, I got the current UTC timestamp `date +%s` on my system, subtracted 2000 and 1000 seconds respectively for `start` and `end`, got a few transactions I'd mined over the past half hour or so